pkg/database/migrations/catalog/20230830134500_seed_db.go

```diff
@@ -14,7 +15,18 @@ var f embed.FS
 func init() {
 	Migrations.MustRegister(func(ctx context.Context, db *bun.DB) error {
 		fixture := dbfixture.New(db)
-		return fixture.Load(context.Background(), f, "testdata.yaml")
+		if err := fixture.Load(ctx, f, "testdata.yaml"); err != nil {
```

Comment:
@theSuess, this adds three initial “fixture” pizzas in Postgres, but it doesn’t update the auto-incrementing id sequence.

As a result, the first three calls to generate pizzas fail because their ids (1, 2, and 3) are already present in the DB.

This change fixes that by updating the sequence in Postgres.

Feel free to tweak this if you know a better or more idiomatic way to handle it.